packages/amplify-category-function/src/index.ts

```diff
@@ -111,10 +110,28 @@ export async function initEnv(context) {
   await sequential(functionTasks);
 }
 
-export function invoke(options) {
-  invokeFunction(options);
+// returns a function that can be used to invoke the lambda locally
+export async function getInvoker(context: any, resourceName: string): Promise<(request: InternalInvocation) => Promise<any>> {
+  const breadcrumbs: FunctionBreadcrumbs = context.amplify.readBreadcrumbs(context, categoryName, resourceName);
+  const runtimeManager: FunctionRuntimeLifecycleManager = await context.amplify.loadRuntimePlugin(context, breadcrumbs.pluginId);
+
+  const lambdaConfig = getAllLambdaFunctions(context, context.amplify.pathManager.getBackendDirPath()).find(
```

Comment:
don't we want an assert here in case if the resource was not found? wrong lambda name or something else could be a reason?